crates/pathfinder/src/bin/pathfinder/main.rs

```diff
@@ -320,7 +320,7 @@ If you are trying to setup a custom StarkNet please use '--network custom',
                 Err(err) => tracing::error!("Cairo process ended unexpected; failed to join task handle: {:?}", err),
             }
         }
-        _result = rpc_handle => {
+        _result = rpc_handle.stopped() => {
```

Comment:
Does this return a more meaningful error now that we can emit on crash?

Previously just returned `()` which is of course useless.

Reply:
I'm afraid it's still the same case now, i.e. `()` .